managed/services/telemetry/config.default.yml

```diff
@@ -348,6 +356,14 @@ telemetry:
       - metric_name: "pmm_server_usage_environments_count"
         column: "environments_count"
 
+  - id: PMMUsagePerconaRolesCount
```

Comment:
nit: this is a compound noun, and, if we want to use the rules, it has to be in singular vs plural, i.e. `PMMUsagePerconaRoleCount`.

Reply:
good point. For consistency purposes I'd opt for keeping it this way, since we use plural in other places.

Reply:
Do we need consistency when it increases the number of things that look weird and don't comply with language rules?

Reply:
I see there's a strong preference in the team for consistency over correctness.
Happy to rename this if there's consensus though.

Reply:
Actually, there is no consistency in applying that particular rule. A few examples fro the code: `scheduleService`, `backupService`, `removalService` are respecting the rules even though they also refer to multiples of schedules, backups, etc.

That said, making it right will not affect any consistency because there is none :)

Reply:
I was referring to other metrics, where we already use plural
- `PMMUsageEnvironmentsCount`
- `PMMUsageServicesCount`
- `PMMUsageNodesCount`
- etc.

Reply:
Sure, I saw that as well... All I'm saying is that getting it right doesn't break anything.

Reply:
Just an extract, no offense :)

"A tree that has apples has many apples, but we say an apple tree, not apples tree; matchbox not matchesbox; toothbrush not teethbrush."